pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -124,8 +124,6 @@ def pre_migrate_content_detail(cls, content_batch):
             *cls.unit_fields)
         pulp2rpm_to_save = []
         for rpm in pulp2_content_batch:
-            rpm['repodata'] = decompress_repodata(rpm['repodata'])
-
             pulp2rpm_to_save.append(
```

Comment:
This makes a lot of sense, it also has the benefit that decompression will happen at publish time, so the parallelism can help with it.

Reply:
@goosemania LGTM pending data migration / final set of change

Reply:
@dralley , I believe it happens earlier, at content migration time. We use repodata from pulp 2 to create a Package instance.

Reply:
Derp, yeah, I think I realized that at some point and then promptly forgot it.